api/apps/api/test/proxy.vector-tiles.e2e-spec.ts

```diff
@@ -100,7 +105,11 @@ describe('ProxyVectorTilesModule (e2e)', () => {
      * https://www.figma.com/file/hq0BZNB9fzyFSbEUgQIHdK/Marxan-Visual_V02?node-id=2991%3A2492
      */
     describe('Admin-areas layers', () => {
-      test.todo('we should test that the response is a valid mvt');
+
+      test.todo('We should test that the response is a valid mvt');
+      test.todo('We should add test for user upload planning units');
+      test.todo('We should add test for irregular planning units');
```

Comment:
nothing to do here, but you could even write the actual test statement (the "we should test..." part is already implicit in the `.todo()` part.